src/stores/utilities/pacbioLibraries.js

```diff
@@ -0,0 +1,45 @@
+const libraryPayload = ({
+  id,
+  pacbio_request_id,
+  template_prep_kit_box_barcode,
+  tag_id,
+  concentration,
+  volume,
+  insert_size,
+}) => {
+  const requiredAttributes = {
+    template_prep_kit_box_barcode,
+    tag_id,
+    concentration,
+    volume,
+    insert_size,
+  }
+
+  return id
```

Comment:
Another way of writing this would be define the shared structure in a variable and then add a conditional after which adds the id-relevant data. Might prevent some duplication but not required.

Reply:
I did think about that but wasn't sure. If it was just the id that would be easy but I got a bit stuck with how to add the pacbio_request_id into the attributes. If you have a solution let me know. I am probably missing the obvious.